libs/cimit-service/src/main/java/uk/gov/di/ipv/core/library/service/CiMitService.java

```diff
@@ -51,7 +51,8 @@
 
 public class CiMitService {
     private static final Logger LOGGER = LogManager.getLogger();
-    private static final Gson gson = new Gson();
+    private static final ObjectMapper objectMapper = new ObjectMapper();
```

Comment:
This should really be `OBJECT_MAPPER`, as it's static final (i.e. constant) - I know the previous one isn't!